db/empty.txt

```diff
@@ -0,0 +1 @@
+So we can commit the directory, but ignore the db inside of it.
```

Comment:
Maybe some day we will actually need something here ACS this can go away.

Perhaps a README for setting up the db.